.github/workflows/php-ci.yaml

```
@@ -106,6 +106,25 @@ jobs:
     - name: Upload coverage to Codecov
       uses: codecov/codecov-action@v3
```

actionlint (1.7.12)

[error] the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)